asreview/state/__init__.py

```diff
@@ -17,3 +17,6 @@
 from asreview.state.hdf5 import HDF5State
 from asreview.state.json import JSONState
 from asreview.state.utils import open_state
+from asreview.state.utils import states_from_dir
```

Comment:
These are generally only used internally I think, but it's fine either way with me.

Reply:
I think extensions are easier to maintain if they make use of the public interface. That's why I requested this.